lib/datadog/tracing/contrib/rack/middlewares.rb

```diff
@@ -219,13 +226,13 @@ def configuration
             Datadog.configuration.tracing[:rack]
           end
 
-          def parse_request_headers(env)
+          def parse_request_headers(headers)
             {}.tap do |result|
```

Comment:
Similarly to above, looks like a job for `each_with_object`

Reply:
I was trying to modify this function as little as possible while still taking advantage of the new class I added :)